### DISCUSSION TITLE
Custom format to captions

### DISCUSSION THREAD

**Discussion by nipnipj** [quarto-dev/quarto-cli#2662] [Q&A]
How can we give a custom format `#| fig-cap:`?

**Comment by mcanouil** [comment]
Since your question is not really "general", could you please move it to the "Q&A" category (using GitHub on a desktop browser)?

You need to provide more information/context if you want to get an answer.
What are you calling "custom format"?

Is the following example from the documentation what you want? <https://quarto.org/docs/computations/r.html#chunk-options>
`#| fig-cap: !expr paste("Air", "Quality")`

**Comment by nipnipj** [comment]
Bold and italic on selected words/expression will suffice.

**Reply by cderv** [reply]
You should be able to use Markdown for that. I believe this is supported, or should be. Did you try it already ? 

````markdown
---
title: Test
---

```{r}
#| fig-cap: I will be **bold** or _emphasize_
plot(cars)
```
````

**Comment by nipnipj** [comment]
Does it work for `#| tbl-cap:` in a `gt` table?

**Reply by mcanouil** [reply]
Best/shorter way to find out, wouldn't it be to try it?

**Reply by nipnipj** [reply]
It is not working for me.

**Reply by cderv** [reply]
No for `tbl-cap` it is not working as expecting in all tables - related issue https://github.com/quarto-dev/quarto-cli/issues/2573
